Models/Item.cs

```diff
@@ -78,6 +78,7 @@ public Dimensions Size
         public string Name { get; set; }
         public string Description { get; set; }
         public int KarmaPoints { get; set; }
+        public String GivenTo { get; set; }
```

Comment:
Is this string will be used just to hold userid, and after actual give, item will be deleted from database? Or the items are not being deleted from db? Maybe could you explain a bit on this? :)

Reply:
Ah, okay. Understood :)